README.md

```diff
@@ -1,2 +1,260 @@
+c can be any complex number, as long as it does not change during recursion. For each different c there is a different set o z's that follow the described rule.\
+The set is named after the French mathematician Gaston Julia
+
+➕ It has fractal (Hausdorff) dimension of:
+
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;D=2"/>
+
+🎨 The Julia set can't be plotted as one single image, due to the infinite possible values of c. Because of that, you must first choose a value of c you want to plot for.\
+The rest of the process is essentialy the same as the one used to plot the Mandelbrot set: by transforming complex numbers to pixel coordinates, you can iterate f<sub>c</sub> for each complex number corresponding to each pixel on the image and color it based on the number of iterations it takes to diverge.\
+Pixel coordinates to plot the Julia fractals should also be normalized to no exceed |2| on any axis, just like with the Mandelbrot set.\
+By changing the complex c chosen, you can get different images. Interestingly, if c ∈ Mandelbrot set, the corresponding Julia fractal will be one solid shape (refer to screenshot 2), and if c ∉ Mandelbrot set, the corresponding Julia fractal will not be just one solid shape, but a collection of disjointed shapes (refer to screenshot 1).
+
+## 3D Fractals
+
+### Sierpinski Tetraheadron
+
+![gif](./media/sierpinski_tetrahedron.gif)
+
+📌 This fractal is the 3-dimensional analogue to the Sierpinski Triangle. It is also known as the Tetrix.
+
+➕ It has fractal (Hausdorff) dimension of:
+
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;D=\log_{2}{4}=2"/>
+
+🎨 The construction process for a Tetrix is very similar to that of the Sierpinski Triangle:
+1. Start with a tetraheadron of side-length L.
+2. Shrink it to half its original height.
+3. Put together four copies of the half-height tetraheadron, so that each of them touch each other in one vertex.
+4. Repeat steps 2 and 3 for the new shapes each iteration.
+
+The first iteration of this loop will give you a level-1 tetraheadron, the second one will give you a level-2 tetraheadron, and so on. The Sierpinski Tetraheadron itself is the resulting shape after infinite iterations.
+
+A similar fractal can be constructed using a square-based pyramid as its base.
+
+### Menger Sponge
+
+![gif](./media/menger_sponge.gif)
+
+📌 This fractal is a three-dimensional generalization of the two-dimensional Sierpinski Carpet, and, likewise, also a generalization of the one-dimensional Cantor Set. It was first described by Karl Menger in 1925, during his studies of topological dimnesnions.
+
+➕ It has fractal (Hausdorff) dimension of:
+
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;D=\log_{3}{20}=2,7268..."/>
+
+🎨 You can construct the Menger Sponge with the following steps:
+1. Start with a cube.
+2. Subdivide each face of the cube in 9 equal sized suqares, like in a Rubik's Cube. This will subdivide the cube in 27 smaller cubes.
```

Comment:
```suggestion
2. Subdivide each face of the cube in 9 equal sized squares, like in a Rubik's Cube. This will subdivide the cube in 27 smaller cubes.
```